### DISCUSSION TITLE
SuperGrate profile backup and restore

### DISCUSSION THREAD

**Discussion by LUS-Mark** [belowaverage-org/SuperGrate#111] [Q&A] [answered]
I just started looking at this to move profiles to new PCs so maybe this is documented somewhere, but is there a way to make it create the .mig file directly to the network store and not do the scanstate locally?  We have instances with large profiles, namely lots of PST files, and there isn't going to be enough free space to save it locally.  Many of the users will be remote as well.

I can run scanstate from the cmd line like scanstate \\server\usmt for example and it will make it on the server but if I try to add that into the scanstate parameters in supergrate and press start I get an error saying the cli properties are incorrect.

**Comment by krisdb2009** [comment, marked as answer]
It might be slow when you do this but you should be able to put a UNC path in this setting:

![image](https://github.com/belowaverage-org/.github/assets/8226974/e45c5416-330d-462e-a155-20b6c7aae44e)

**Reply by LUS-Mark** [reply]
I was going to try that, yes its slower this way but in the case of an insuffcient disk space it would be the only option.  

Also - is the OneDrive folder excluded by default?

**Reply by krisdb2009** [reply]
I believe so.
